File: chatgpt-generated/go/ssrf.go
package main

import (
	"io"
	"net/http"
)

func main() {
	http.HandleFunc("/fetch", func(w http.ResponseWriter, r *http.Request) {
		url := r.URL.Query().Get("url")
		resp, err := http.Get(url)
		if err != nil {
			w.Write([]byte("Error fetching URL"))
			return
		}
		defer resp.Body.Close()
		io.Copy(w, resp.Body)
	})
	http.ListenAndServe(":8080", nil)
}

File: chatgpt-generated/python/ssrf.py
from flask import Flask, request
import requests

app = Flask(__name__)

@app.route('/fetch')
def fetch():
    url = request.args.get('url', '')
    try:
        r = requests.get(url)
        return r.text
    except:
        return "Error fetching URL"

if __name__ == '__main__':
    app.run(debug=True)
